File: bench/perf/java/Wordcount.java
import java.util.*;

public class Wordcount {
    public static void main(String[] args) {
        String seed = "the quick brown fox jumps over the lazy dog ";
        String text = seed.repeat(500000);
        String[] words = text.split(" ", -1);
        Map<String, Integer> counts = new HashMap<>();
        for (String w : words) counts.merge(w, 1, Integer::sum);
        System.out.println(counts.size());
        System.out.println(counts.get("the"));
    }
}

File: bench/perf/go/wordcount.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	seed := "the quick brown fox jumps over the lazy dog "
	text := strings.Repeat(seed, 500000)
	words := strings.Split(text, " ")
	counts := map[string]int{}
	for _, w := range words {
		counts[w]++
	}
	fmt.Println(len(counts))
	fmt.Println(counts["the"])
}
